MFVNeutralino/python/VertexAuxProducer_cfi.py

```diff
@@ -7,14 +7,22 @@
                                    kvr_params = kvr_params,
                                    beamspot_src = cms.InputTag('offlineBeamSpot'),
                                    primary_vertex_src = cms.InputTag('goodOfflinePrimaryVertices'),
-                                   muons_src = cms.InputTag('selectedPatMuons'),
-                                   electrons_src = cms.InputTag('selectedPatElectrons'),
                                    gen_vertices_src = cms.InputTag('mfvGenParticles', 'decays'),
                                    vertex_src = cms.InputTag('mfvVertices'),
+                                   vertex_seed_tracks_src = cms.InputTag('mfvVertices', 'seed'),
                                    sv_to_jets_src = cms.string('dummy'),
                                    track_ref_getter = jmtTrackRefGetter,
+                                   sv_to_muons_src = cms.string('dummy'),
+                                   sv_to_ele_src = cms.string('dummy'),
+                                   muons_src = cms.InputTag('selectedPatMuons'),
+                                   electrons_src = cms.InputTag('selectedPatElectrons'),
+                                   rho_src = cms.InputTag('fixedGridRhoFastjetAll'),
+                                   electron_effective_areas = cms.FileInPath('RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_92X.txt'),
                                    sort_by = cms.string('ntracks_then_mass'),
                                    verbose = cms.untracked.bool(False),
                                    )
 
-mfvVerticesAux = mfvVerticesAuxTmp.clone(sv_to_jets_src = 'mfvVerticesToJets')
+mfvVerticesAux = mfvVerticesAuxTmp.clone(sv_to_jets_src = 'mfvVerticesToJets',
+                                         sv_to_muons_src = 'mfvVerticesToLeptons',
+                                         sv_to_ele_src = 'mfvVerticesToLeptons'
```

Comment:
Same question as before for @abbywarden: does this introduce any changes to how vertices are reconstructed in the non-displaced-vertex analysis?